pydarn/plotting/rtp.py

```diff
@@ -335,11 +340,21 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
                                      start_time=start_time,
                                      end_time=end_time,
                                      opt_beam_num=cls.dmap_data[0]['bmnum'])
-        if coord is Coord.SLANT_RANGE:
+                   
+        if coord is Coords.SLANT_RANGE:
             # Get rxrise from hardware files (consistent with RST)
             rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
                                     .hardware_info.rx_rise_time
             y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+        elif coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
+            rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
+                                    .hardware_info.rx_rise_time
+            y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+            Re = 6371
+            y = gateGroundScatter(y, Re, hgt)
+            y0inx = np.min(np.where(np.isfinite(y))[0])
```

Comment:
Either comment or a better variable name. I assume zeroth y index? Which I am not sure what this is used for. 
or is it minimum y value?